README.md

```diff
@@ -29,16 +29,16 @@ Or install it yourself as:
 
 `Avromatic` supports the following configuration:
```

Comment:
I like keeping the configuration mostly in one place, so I'm first attempting to add more structure to this section. We can see how that works.